src/com/owncloud/android/lib/resources/activities/GetRemoteActivitiesOperation.java

```diff
@@ -59,7 +59,7 @@
     private static final String TAG = GetRemoteActivitiesOperation.class.getSimpleName();
 
     // OCS Routes
-    private static final String OCS_ROUTE = "/apps/activity/api/v2/activity";
+    private static final String OCS_ROUTE = "/ocs/v2.php/apps/activity/api/v2/activity";
```

Comment:
Yes, @mario @MorrisJobke it's on my todo for tomorrow to finish the PR:
https://github.com/nextcloud/activity/pull/125